src/days-2-4-adv/adv.py

```diff
@@ -49,3 +59,31 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+# DIRECTION
+
+def direction(direction, current):
+    # Try to move a direction, or print error if player can't go that way.
+    # Returns room the player moved to or same room if player didn't move.
+    attrib = direction + '_to'
+    # See if the room has the destination attribute
+    if hasattr(current, attrib):
+        # If so, return its value (the next room)
+        return getattr(current, attrib)
+    # Otherwise print an error and stay in the same room
+    print("You can NOT go that way")
+    return current
+
+done = False
```

Comment:
Is there a way to avoid this global variable? Global variables, especially mutable ones, can be dangerous.